### ISSUE TITLE
Airline Tabline Redraw Bug?

### ISSUE THREAD

**Issue by starcraftman** [junegunn/goyo.vim#63]
I'm not sure who is responsible but only manifests after Goyo command, may require upstream fix in airline or even a vim redraw bug?

Test: Use :Goyo twice, then you'll see a graphical bug on separators of the tabline. I.e. where the > separates colors. Then put hack into config & it won't reproduce.

Test config to reproduce:
```vim
let g:win_shell = (has('win32') || has('win64')) && &shellcmdflag =~ '/'
let g:cygwin_shell = has('win32unix')
let g:vim_dir = g:win_shell ? '$HOME/vimfiles' : '$HOME/.vim'

try
  call plug#begin(expand(g:vim_dir . '/bundle'))

  Plug 'bling/vim-airline'
  Plug 'junegunn/goyo.vim'
  Plug 'tomasr/molokai'
  
  call plug#end()
catch
  echomsg 'Run command `Bootstrap`. After restart run command `pi'
endtry

let g:airline#extensions#syntastic#enabled = 1

let g:airline#extensions#tabline#enabled = 1

let g:airline_powerline_fonts = 1

let &statusline  = '%<%1* %t%m %y || %{&fenc?&fenc:&enc}[%{&ff}] ' 
let &statusline .= '%= %b(0x%B) || buf: %n || %p%% %l,%c %0*'
set laststatus=2

syntax off
set background=dark
colorscheme molokai
syntax on
```

Hack that resolves problem:
```vim
function s:goyo_leave()
  split
  redraw
  quit
endfunction

autocmd! User GoyoLeave
autocmd  User GoyoLeave nested call <SID>goyo_leave()
```

**Comment by starcraftman** [comment]
Picture: ![tab](https://cloud.githubusercontent.com/assets/470400/6855219/bf51494c-d3cc-11e4-83e5-0ecfeec61cb0.png)

**Comment by junegunn** [comment]
I don't use airline, so, can you check if the problem is reproducible without Goyo? Goyo toggles airline with the following code:

- https://github.com/junegunn/goyo.vim/blob/master/autoload/goyo.vim#L206
- https://github.com/junegunn/goyo.vim/blob/master/autoload/goyo.vim#L340-L341

**Comment by starcraftman** [comment]
@junegunn I did think of that during my own debugging. Without goyo.vim the AirlineToggle command redraws fine. I've noticed previously that sometimes airline doesn't handle resizes well. I think it has more to do with other stuff modified during the setup that airline expects maybe not be changed? Will try to investigate further. 

Even if I don't find root cause, the above solution may help others meantime.
